### PR TITLE
Tickets/dm35314: Converted to pyproject.toml

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+name = "ts_SalMultiLanguageTests"
+description = "SAL tests to verify that publishers and subscribers can communicate between all the programming languages, C++, Java and SalObj."
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = { documentation = "https://jira.lsstcorp.org/secure/Dashboard.jspa", repository = "https://github.com/lsst-ts/ts_SalMultiLanguageTests" }
+dynamic = [ "version" ]
+dependencies = []
+
+[tools.setuptools]
+package-data = {"" = "*.csv"}
```
Comment (wvreeven) [inline pyproject.toml:15]: This package doesn't contain any csv files so please remove this section.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+[tool.setuptools.dynamic]
+version = { attr = "setuptools_scm.get_version" }
+
+[tool.setuptools.packages.find]
+where = [ "python" ]
+
+[project.scripts]
```
Comment (wvreeven) [inline pyproject.toml:23]: Please remove this section since it is empty.

File: setup.py [hunk 3]
```diff
@@ -1,26 +1,28 @@
-import setuptools
+#!/usr/bin/env python
+#
+# This file is part of ts_IntegrationTests.
```
Comment (wvreeven) [inline setup.py:3]: `ts_IntegrationTests` -> `ts_SalMultiLanguageTests`

Please check the other Python files as well.

**Review by wvreeven — APPROVED** [review]
Looks good to me. I have added a few comments for some small possible improvements. I leave it to you to process or ignore them.

File: setup.py (hunk 3 above)
Comment (rbovill) [reply setup.py]: Fixed

File: pyproject.toml (hunk 2 above)
Comment (rbovill) [reply pyproject.toml]: Removed.

File: pyproject.toml (hunk 1 above)
Comment (rbovill) [reply pyproject.toml]: Removed.

**Pull request merged**